TOC.md

```diff
@@ -509,11 +509,11 @@
     + [参考手册](/tiup/tiup-reference.md)
     + [TiDB 集群拓扑文件配置](/tiup/tiup-cluster-topology-reference.md)
     + [TiUP 镜像参考指南](/tiup/tiup-mirror-reference.md)
-    + TiUP 组件文档
-      + [tiup-playground 运行本地测试集群](/tiup/tiup-playground.md)
-      + [tiup-cluster 部署运维生产集群](/tiup/tiup-cluster.md)
-      + [tiup-mirror 定制离线镜像](/tiup/tiup-mirror.md)
-      + [tiup-bench 进行 TPCC/TPCH 压力测试](/tiup/tiup-bench.md)
+    + TiUP 支持场景
+      + [使用 tiup-playground 运行本地测试集群](/tiup/tiup-playground.md)
+      + [使用 tiup-cluster 部署运维生产集群](/tiup/tiup-cluster.md)
+      + [使用 tiup-mirror 定制离线镜像](/tiup/tiup-mirror.md)
+      + [使用 tiup-bench 进行 TPCC/TPCH 压力测试](/tiup/tiup-bench.md)
```

Comment:
@lilyjazz How about using the simplified version in the left navigation panel like below? It seems more user friendly?
```suggestion
      + [运行本地测试集群](/tiup/tiup-playground.md)
      + [部署运维生产集群](/tiup/tiup-cluster.md)
      + [定制离线镜像](/tiup/tiup-mirror.md)
      + [进行 TPCC/TPCH 压力测试](/tiup/tiup-bench.md)
```

Reply:
I don't think so. 
Users use TiUP through these components. They can recognize these article purposes by components' names.
On the other hand, we can deploy cluster or make benchmark by different ways, these articles only focus on the TiUP component's way.
Therefore, I suggest use the title with tiup+component.

Reply:
```suggestion
      + [使用 TiUP Bench 进行 TPCC/TPCH 压力测试](/tiup/tiup-bench.md)
```